### PR TITLE
About page left me broken- development I was through and hopin'

### PR DESCRIPTION
**Pull request by lilcheerio**
And even though it's not Wednesday my empty arms were open. But today was Thursday, and Max's OtterPilot was waiting for love, waiting for love.
THANK OUR BRAINS ITS FRIDAY!

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -1,261 +1,319 @@
 <!DOCTYPE html>
 <html lang="en">
-	<!-- This file is a placehodler for a page. Rename this file to the name of the page you are working on. -->
+<!-- This file is a placehodler for a page. Rename this file to the name of the page you are working on. -->
```
Comment (SolivanLau) [inline about.html]: I think we can remove this comment!

File: about.html [hunk 2]
```diff
@@ -1,261 +1,319 @@
...
-		<!-- done -->
-		<link rel="stylesheet" href="/css/about.css" />
-	</head>
+<head>
+	<meta charset="UTF-8" />
+	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+	<title>If These Lands Could Talk - Page Name</title>
+	<link rel="preconnect" href="https://fonts.googleapis.com" />
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+	<link href="https://fonts.googleapis.com/css2?family=Josefin+Sans:wght@400;500;700&display=swap" rel="stylesheet" />
```
Comment (SolivanLau) [inline about.html:10]: Looks good, I believe there is another `<link>` with a url for the open sans that needs to be added in for the body text. If you are still waiting for it, please let me know and we can add this in later

File: about.html [hunk 3]
```diff
@@ -1,261 +1,319 @@
...
+	<link rel="stylesheet" href="/css/normalize.css" />
+	<link rel="stylesheet" href="/css/global.css" />
+	<!-- In a new branch, rename this page.css file to the name of that actual page (Example: home.css) -->
```
Comment (SolivanLau) [inline about.html]: Please feel free to remove all comments for instructions for the page template

File: about.html [hunk 4]
```diff
@@ -1,261 +1,319 @@
...
+			<picture>
+				<source media="(min-width:768px)" srcset="/assets/images/pages/about/hero-about.jpg" />
+				<img src="/assets/images/pages/about/mobile/hero-about-mobile.jpg" alt="woman meditating"
+					style="width: 100%" />
+			</picture>
+		</div>
+
+		<div class="invis-flex">
+			<div class="hero-text">
+				<h1 class="h3-styles">
```
Comment (SolivanLau) [inline about.html:126]: I am noticing an issue from 320px - 380px in mobile view where the `<h1>` is not adjusting to the device width. Please make sure that the `<h1>` styles are working properly in **about.css**:

image

File: css/global.css [hunk 5]
```diff
@@ -332,6 +335,256 @@ Global Component Styles (buttons, forms, etc.)
 	color: var(--color-onyx);
 }
 
+/* HEADER STYLES BB */
+/*
+===================================
+===================================
+Styles for an individual page
```
Comment (SolivanLau) [inline css/global.css:342]: Please feel free to remove these instruction comments! just leave the main heading comments for easier readability

**Review by SolivanLau — CHANGES_REQUESTED** [review]
Hello thank you for your hard work! just a few changes are requested overall! Here's a quick recap:

1. remove any instruction comments throughout **about.html** and **about.css** for easier readability
2. there is an responsive issue for the `<h1>` element in the about page (320px - 380px)
3. bring over the **open sans ** link to **about.html**
4. there is a merge conflict in your PR, please address before rereview :)

**Review by SolivanLau — CHANGES_REQUESTED** [review]
everything looks great! One tiny request: can you please update the Page Name in the  `<title>` element in **about.html**  

change from Page Name to About us, thank you!

**Review by maxharrisnet — APPROVED** [review]

**Review by SolivanLau — APPROVED** [review]
Thank you!

**Pull request merged**